cmd/openshift-install/create.go

```diff
@@ -488,6 +500,67 @@ func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset
 	return nil
 }
 
+// When bootstrap on SNO deployments, we should not remove the bootstrap node prematurely,
+// here we make sure that the deployment is stable.
+// Given the nature of single node we just need to make sure things such as etcd are in the proper state
+// before continuing.
+func waitForStableSNOBootstrap(ctx context.Context, config *rest.Config) error {
+	timeout := 5 * time.Minute
+
+	// If we're not in a single node deployment, bail early
+	if assetStore, err := assetstore.NewStore(command.RootOpts.Dir); err == nil {
```

Comment:
Maybe we can move it to function ? and make it reusable for future generations?
Something like IsSno?

Reply:
Yeah, that's a good point, I wasn't sure where such a function would live though, I put it in the same file for now, but I'm open to moving it where it makes more sense.